sroka/api/sparql/sparql.py

```diff
@@ -0,0 +1,36 @@
+from SPARQLWrapper import SPARQLWrapper, JSON
+from sroka.api.sparql.sparql_helpers import get_options_from_config
+from configparser import NoSectionError
+from sroka.api.helpers import save_to_file
+import pandas as pd
+
+
+def query_sparql(query, endpoint_url=None, filename=None):
+    try:
+        options = get_options_from_config()
+
+    except NoSectionError:
+        print('Missing MySQL section in configuration')
+        return pd.DataFrame([])
+    if endpoint_url:
+        options['endpoint_url'] = endpoint_url
+
+    sparql = SPARQLWrapper(endpoint_url)
```

Comment:
I believe here we should pass options["endpoint_url"], so it works when endpoint is passed only in config @martynaut

Reply:
Fixed. Thank you!